pkgs/tools/misc/fpp/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, bzip2, python27Full, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  version = "0.7.1";
+  name = "fpp-${version}";
+
+  src = fetchFromGitHub {
+    owner = "facebook";
+    repo = "PathPicker";
+    rev = "refs/tags/${version}";
+    sha256 = "1mfyr9k5s3l1sg3c9vlyiqg8n1wwppzb981az2xaxqyk95wwl1sa";
+    name = "fpp-${version}";
+  };
+
+  buildInputs = [ bzip2 makeWrapper ];
+
+  buildPhase = ''
+    mkdir -p "$out/bin"
+    mv "fpp" "src" "$out"
+    ln -s "$out/fpp" "$out/bin/fpp"
+  '';
+
+  installPhase = ''
```

Comment:
Instead of a wrapper, I think it'd be much better to patch the script directly. Looking at the source, it sets `PYTHONCMD="python"` so that's probably what you want to patch.